core/domain/state_domain.py

```diff
@@ -3257,6 +3257,26 @@ def _get_all_translatable_content(self):
 
         return content_id_to_translatable_item
 
+    def has_content_id(self, content_id):
+        """Returns whether a given content ID is available in the translatable
+        content.
+
+        Args:
+            content_id: str. The content ID that needs to be checked for the
+            availability.
+
+        Returns:
+            bool. A boolean that indicates the availability of the content ID
+            in the translatable content.
+        """
+        is_content_id_available = False
```

Comment:
Don't know why you need this -- just get the translatable content and then check to see if content_id is in it.

I.e. 

```
        available_translate_content = self._get_all_translatable_content()
        return bool(content_id in available_translate_content)
```

or similar.

Reply:
Done